frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -986,7 +986,7 @@ function ReaderHighlight:onShowHighlightDialog(index)
             {
                 text = "…",
                 callback = function()
-                    self.selected_text = util.tableDeepCopy(item)
+                    self.selected_text = { text = item.text, pos0 = item.pos0, pos1 = item.pos1 }
```

Comment:
Somehow the same here.
How often is this happening, and do we really need to tavoir the tableDeepCopy() ?

Reply:
I believe that even small steps to speed-up and to reduce memory usage are valuable.